wordpress-mobile/WordPress-iOS
# 5537 insight stats preloading

Addresses part of #5537

To test:
1. While on wifi, go to a site's insight stats screen. The data should be preloaded after selecting the site, and take no time to load data on the insights screen.
2. While on 3G or LTE, there should be no preloading. The insight stats screen should only load stats once opened.

Needs review: @bummytime
